geniza/footnotes/admin.py

```diff
@@ -28,6 +29,53 @@ class AuthorshipInline(SortableInlineAdminMixin, admin.TabularInline):
     extra = 1
 
 
+class SourceFootnoteInline(admin.TabularInline):
+    """Footnote inline for the Source admin"""
+
+    model = Footnote
+    fields = (
+        "object_link",
+        "content_type",
+        "object_id",
+        "location",
+        "doc_relation",
+        "notes",
+    )
+    readonly_fields = ("object_link",)
+
+    def object_link(self, obj):
+        """edit link with string display method for associated content object"""
+        # return empty spring for unsaved footnote with no  content object
+        if not obj.content_object:
+            return ""
+        content_obj = obj.content_object
+        edit_url = "admin:%s_%s_change" % (
+            content_obj._meta.app_label,
+            content_obj._meta.model_name,
+        )
+        edit_path = reverse(edit_url, args=[obj.object_id])
+        return format_html(
+            f'<a href="{edit_path}">{content_obj} '
+            + '<img src="/static/admin/img/icon-changelink.svg" alt="Change"></a>'
+        )
+
+    object_link.short_description = "object"
```

Comment:
When extra is not defined, it defaults to 3. I prefer fewer, even 0, but I don't think this matters too much right?

Reply:
I set it to 0 for some other things in admin, fwiw

Reply:
The whole point of this story is for them to easily add more footnotes for a single source, so we _want_ extra footnotes — I thought three was a reasonable number.

Reply:
that makes sense, getting into the weeds, it's easy to forget what we're actually trying to accomplish!

Reply:
For forms where we expect them to regularly add more related data, I don't think we should set it to zero, but I'm ok with setting it to a smaller number especially if the change form is already long (like Document)